Makefile

```diff
@@ -17,6 +17,7 @@ libnavui-resources \
 libnavui-voice \
 libnavigation-android \
 libnavui-speedlimit \
+libnavui-car \
```

Comment:
Similar to `CORE_MODULES` and `UI_MODULES` what do you think about adding a `CAR_MODULE`. It probably doesn't make sense to run the complete set of UI modules for every task whenever a change to a file in car module is made.

Reply:
I thought about this. But I don't think we should split them up here, because it's an optimization that we don't know the impact and it would introduce a lot of redundancy. Everywhere `UI_MODULES` is called we need a separate `CAR_MODULE`. 

In terms of local development, running `./gradlew libnavui-car:ktlint` works just fine. So I'm assuming the concern would be `CI/CD` flows like `circleci`, but we probably have some better optimization to make there.

Reply:
I actually do like @abhishek1508's idea. It would allow us to run checks on CI with just a few modifications to build scripts and also be more granular with distribution by having a new job:
```
- publish-artifacts:
    artifact-type: "car"
```
that can build snapshot but not regular releases.

Reply:
> ````
> - publish-artifacts:
>     artifact-type: "car"
> ```

But isn't this the same as `./gradlew libnavui-car:mapboxSDKRegistryUpload`?

I'll dig in to see, because I'm not seeing why we would want it